src/Compilers/Core/MSBuildTask/Vbc.cs

```diff
@@ -294,6 +294,19 @@ internal void MovePdbFileIfNecessary(string outputAssembly)
                 return;
             }
 
+            // Design-time builds should not have side effects.
```

Comment:
There doesn't appear to be a Task method that gets called when build completes successfully. Ideally, this issue should be fixed by adding a /pdb:file option to the VB compiler (see #3859) and removing the handling of .pdbs here. If we make a fix here in the meantime, the fix should be relatively simple.